docs/docs/en/architecture/configuration.md

```diff
@@ -162,9 +162,9 @@ The default configuration is as follows:
 
 Note that DolphinScheduler also supports database configuration through `bin/env/dolphinscheduler_env.sh`.
 
-### Zookeeper related configuration
+### Registry related configuration
```

Comment:
I think we should capitalize every word of a sub-title. `Registry Related Configuration`